static/js/robot.js

```diff
@@ -586,3 +980,43 @@ this.onmessage = function(e) {
         }
     }
 }
+
+function FieldObj() {
```

Comment:
We are adding new classes to separate files now. Make a new file static/js/FieldObj.js and move the FieldObj and Wall classes there. Add a line `importScripts(./FieldObj.js)` near the top (with the other calls to importScripts).